### ISSUE TITLE
About handleSignUpFormSubmit function

### ISSUE THREAD

**Issue by YCyoon1994** [swsnu/swppfall2021#27]
I have a question about the birth date format. 
If we use the regex expression to validate the date format, some invalid cases could occur.
Such as 2011-11-31. Is it a valid birthdate format in this homework?

**Comment by kdh0102** [comment]
Yes, the example you wrote (`2011-11-31`) will be treated as a valid format.
You can assume that the date can go up to 31 for each month.

**Comment by YCyoon1994** [comment]
Thank you 👍
